### ISSUE TITLE
[Feature Request] force screensaver brightness to 0 on not_home

### ISSUE THREAD

**Issue by Chreece** [joBr99/nspanel-lovelace-ui#78]
Is there any way to have some options like brightness to be changed on demand?
I think an option like: to turn the brightness to 0 when no one is in the room or at home, would make sense and also be more energy efficient.
Happy to read your thoughts! :)

Greetings from Köln

**Comment by joBr99** [comment]
You can send updates for brightness from nodered or something like this. Just send `dimmode,x` to the custom send topic

**Comment by Chreece** [comment]
> You can send updates for screensaver brightness from nodered or something like this. Just send `dimmode,x` to the custom send topic

So this already can be done! Sorry didn't thought about it thank you!

**Comment by joBr99** [comment]
Guess I will still add a config option for an entity that forces brightness to 0 if the state is equal not_home or something like that. But for dynamic changes from a lux sensor or something like that dimmode is the way to go.

**Comment by Chreece** [comment]
![image](https://user-images.githubusercontent.com/68458228/160236620-e0841764-3e4d-4ad4-b806-ea0cf92f67a3.png)

**Comment by Chreece** [comment]
yes it works! A whole new world has opened to me :)
Is there any way to show non latin letters on the panel? Greek letters doesn't show up even with the babel "el_GR" locale

**Comment by joBr99** [comment]
Nope, isn't supported by the nextion screen, charset that can be transmitted over serial is limited, that's also the reason why I have to use IDs for the Icons.

See the readme in the HMI folder for the serial command's. And also the node red folder :)

**Comment by joBr99** [comment]
With the current development version you should be able to add an device_tracker/person/group to the config, it will update/force the brightness value to 0 if the state changes to "not_home", if state changes back to something else it will update the screen with the currently configured brightness.

```
    brightnessScreensaverTracking: device_tracker.s21_von_johannes_4
```

**Comment by joBr99** [comment]
implemented with release v1.8.3
